content/docs/authorization/deployment/rpm/_index.md

```diff
@@ -130,200 +130,16 @@ If errors occur during installation, review the [Troubleshooting](../../troubles
-  
-  # Specify info about Array2
-  export Array2Type=""
-  export Array2SystemID=""
-  export Array2User=""
-  export Array2Password=""
-  export Array2Pool=""
-  export Array2Endpoint=""
-
-  # Specify IPs
-  export DriverHostVMIP="" 
-  export DriverHostVMPassword=""
-  export DriverHostVMUser=""
-
-  # Specify Authorization proxy host address. NOTE: this is not the same as IP
-  export AuthorizationProxyHost=""
-
-  echo === Creating Storage(s) ===
-  # Add array1 to authorization
-  karavictl storage create \
-            --type ${Array1Type} \
-            --endpoint  ${Array1Endpoint} \
-            --system-id ${Array1SystemID} \
-            --user ${Array1User} \
-			      --password ${Array1Password} \
-            --array-insecure
-  
-  # Add array2 to authorization
-   karavictl storage create \
-            --type ${Array2Type} \
-            --endpoint  ${Array2Endpoint} \
-            --system-id ${Array2SystemID} \
-            --user ${Array2User} \
-			      --password ${Array2Password} \
-            --array-insecure
-    
-  echo === Creating Tenant ===
-  karavictl tenant create -n $TenantName --insecure --addr "grpc.${AuthorizationProxyHost}"
-
-  echo === Creating Role ===
-  karavictl role create \
-           --role=${RoleName}=${Array1Type}=${Array1SystemID}=${Array1Pool}=${RoleQuota} \
-           --role=${RoleName}=${Array2Type}=${Array2SystemID}=${Array2Pool}=${RoleQuota}   
-
-  echo === === Binding Role ===
-  karavictl rolebinding create --tenant $TenantName  --role $RoleName --insecure --addr "grpc.${AuthorizationProxyHost}"
-  ```
-
-### Generate a Token
-
-After creating the role bindings, the next logical step is to generate the access token. The storage admin is responsible for generating and sending the token to the Kubernetes tenant admin.
-
->__Note__: 
-> - The `--insecure` flag is only necessary if certificates were not provided in `$HOME/.karavi/config.json`.
-> - This sample copies the token directly to the Kubernetes cluster master node. The requirement here is that the token must be copied and/or stored in any location accessible to the Kubernetes tenant admin.
-
-  ```
-  echo === Generating token ===
-  karavictl generate token --tenant $TenantName --insecure --addr "grpc.${AuthorizationProxyHost}" | sed -e 's/"Token": //' -e 's/[{}"]//g' -e 's/\\n/\n/g' > token.yaml
-
-  echo === Copy token to Driver Host ===
-  sshpass -p ${DriverHostPassword} scp token.yaml ${DriverHostVMUser}@{DriverHostVMIP}:/tmp/token.yaml 
-  ```
-
-### Copy the karavictl Binary to the Kubernetes Master Node
-
-The karavictl binary is available from the CSM for Authorization proxy server.  This needs to be copied to the Kubernetes master node for Kubernetes tenant admins so the Kubernetes tenant admins can configure the Dell CSI driver with CSM for Authorization.
+The first part of CSM for Authorization deployment is to configure the proxy server. This is controlled by the Kubernetes storage administrator.
```

Comment:
Storage Administrator, not k8s storage administrator.

Reply:
fixed.